kclvm/tools/src/LSP/src/completion.rs

```diff
@@ -336,40 +336,58 @@ fn completion_import_builtin_pkg(
 /// Complete schema attr
 ///
 /// ```no_run
+/// #[cfg(not(test))]
 /// p = Person {
 ///     n<cursor>
 /// }
 /// ```
 /// complete to
 /// ```no_run
+/// #[cfg(not(test))]
 /// p = Person {
 ///     name<cursor>
 /// }
 /// ```
 fn completion_attr(
     program: &Program,
     pos: &KCLPos,
-    prog_scope: &ProgramScope,
+    gs: &GlobalState,
 ) -> IndexSet<KCLCompletionItem> {
-    let mut completions: IndexSet<KCLCompletionItem> = IndexSet::new();
-
-    if let Some((node, schema_expr)) = is_in_schema_expr(program, pos) {
-        let schema_def = find_def(node, &schema_expr.name.get_end_pos(), prog_scope);
-        if let Some(schema) = schema_def {
-            if let Definition::Object(obj, _) = schema {
-                let schema_type = obj.ty.into_schema_type();
-                completions.extend(schema_type.attrs.iter().map(|(name, attr)| {
-                    KCLCompletionItem {
-                        label: name.clone(),
-                        detail: Some(format!("{}: {}", name, attr.ty.ty_str())),
-                        documentation: attr.doc.clone(),
-                        kind: Some(KCLCompletionItemKind::SchemaAttr),
+    let mut items = IndexSet::new();
+    if let Some((_, schema_expr)) = is_in_schema_expr(program, pos) {
```

Comment:
Do we need this function `is_in_schema_expr` with the new semantic and how to ensure the correctness of the function.